test/streaming.test.ts

```diff
@@ -25,14 +25,28 @@ if (isNodeJS()) {
    */
   it('should subscribe to topic, create account, and receive event of account has been created', async () => {
     type Account = { Id: string; Name: string };
+
+    const id = Date.now();
+
+    const accountName = `My New Account #${id}`;
+
+    const pushTopicName = `Topic-${id}`;
```

Comment:
unique `PushTopic` name to avoid duplicate error on test retries (needed now that the push topic is created inside this test).

We try to delete it at the end of the test but it might not if there's a failure before (and we could use a cleanup test helper from jest but these test run on browser so we can't use any jest-specific feature.